lib/ts/recipe/oauth2provider/api/utils.ts

```diff
@@ -126,6 +126,66 @@ export async function loginGET({
     };
 }
 
+export async function logoutGET({
+    logoutChallenge,
+    recipeImplementation,
+    session,
+    userContext,
+}: {
+    logoutChallenge: string;
+    recipeImplementation: RecipeInterface;
+    session?: SessionContainerInterface;
+    userContext: UserContext;
+}) {
+    if (session !== undefined) {
+        const appInfo = SuperTokens.getInstanceOrThrowError().appInfo;
+        const websiteDomain = appInfo
+            .getOrigin({
+                request: undefined,
+                userContext: userContext,
+            })
+            .getAsStringDangerous();
+        const websiteBasePath = appInfo.websiteBasePath.getAsStringDangerous();
+
+        const queryParamsForLogoutPage = new URLSearchParams({
+            logoutChallenge,
+        });
+
+        return {
+            redirectTo: websiteDomain + websiteBasePath + "/oauth/logout" + `?${queryParamsForLogoutPage.toString()}`,
+        };
```

Comment:
Not sure I understand - why is this redirecting?

Reply:
This comment was meant for `logoutPOST`. I have changed the attribute name from `redirectTo` to `frontendRedirectTo`.